tests/scripts/thread-cert/Cert_5_1_02_ChildAddressTimeout.py

```diff
@@ -79,18 +105,20 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[ROUTER].get_state(), 'router')
 
-        self.nodes[ED].start()
+        self.nodes[MED].start()
         self.simulator.go(5)
-        self.assertEqual(self.nodes[ED].get_state(), 'child')
+        self.assertEqual(self.nodes[MED].get_state(), 'child')
 
         self.nodes[SED].start()
         self.simulator.go(5)
         self.assertEqual(self.nodes[SED].get_state(), 'child')
 
-        ed_addrs = self.nodes[ED].get_addrs()
+        self.collect_rlocs()
+
+        ed_addrs = self.nodes[MED].get_addrs()
         sed_addrs = self.nodes[SED].get_addrs()
 
-        self.nodes[ED].stop()
+        self.nodes[MED].stop()
         self.simulator.go(5)
```

Comment:
nit: magic number.
The waiting time here should be related to DEFAULT_CHILD_TIMEOUT
similarly for below sed operations.

Reply:
changed to wait DEFAULT_CHILD_TIMEOUT